webapp/pages/Assessment/AssessmentHome/PanEuropeanHome/panEuropeanHome.less

```diff
@@ -11,12 +11,14 @@
     }
   }
 
-  .partners__imgs {
-    height: 100px;
+  .partners__disclaimer {
+    display: flex;
   }
 
   .partner_che {
-    width: 300px;
+    margin-left: 12px;
+    vertical-align: center;
```

Comment:
you don't need vertical-align here, but align-items in the parent flex container

Reply:
I removed the vertical align to have the logo centered. If I add align-items, the logo itself will be misaligned.
Pic 1: current
![Screenshot 2020-10-27 at 10 45 02](https://user-images.githubusercontent.com/5508251/97277711-7fbd8600-1841-11eb-9bde-d8c374f418d5.png)
Pic 2: align-items: center on parent
![image](https://user-images.githubusercontent.com/5508251/97277790-995ecd80-1841-11eb-9662-cffafee4d9fe.png)